Sources/RInAppMessaging/Utilities/BundleInfo.swift

```diff
@@ -7,12 +7,16 @@ import RSDKUtils
 
 internal class BundleInfo {
 
+    class var bundle: Bundle {
```

Comment:
Does it need to be `class var`? Is it for overriding `bundle`? or can it be `static var`?

Reply:
Yes, I used `class` instead of `static` for overriding/mocking in unit tests

Reply:
Ok 👍